addons/web/models/base_document_layout.py

```diff
@@ -136,7 +141,12 @@ def _compute_preview(self):
 
     def _get_render_information(self, styles):
         self.ensure_one()
-        wizard_with_logo = self.with_context(bin_size=False)
+        # guarantees that bin_size is always set to False,
+        # so the logo always contains the bin data instead of the binary size
+        if self.env.context.get('bin_size'):
+            wizard_with_logo = self.with_context(bin_size=False)
+        else:
+            wizard_with_logo = self
```

Comment:
All this dif is a revert from what was done here https://github.com/odoo/odoo/commit/45d924b7c8e3a2a056a940130591c6bdc0263373, Can you specify in you commit message that.